.github/ISSUE_TEMPLATE/feature_graduation.md

```diff
@@ -16,6 +16,7 @@ about: Plan the graduation of an experimental feature
 
 Checklist to be completed when graduating an experimental feature
 
+- [ ]  Updates for Semantic Kernel Docs updates are complete and checked in here: https://github.com/MicrosoftDocs/semantic-kernel-pr
```

Comment:
Link is not publicly available so add it to the ignore list here: https://github.com/microsoft/semantic-kernel/blob/main/.github/workflows/markdown-link-check-config.json